db/migrate/20200917003811_create_events.rb

```diff
@@ -7,6 +7,7 @@ def change
       t.date 'date', null: false
       t.time 'time'
       t.text 'type'
+      t.boolean 'hidden'
```

Comment:
Do not do this,  this is a painful way to solve the issue. You need to do `rails g migration add_hidden_to_events` and then in the new migrations file do something like:
```rb
add_column :events, :hidden, :boolean
```
Otherwise the migration process will be very painful and we will all have to reset our databases.